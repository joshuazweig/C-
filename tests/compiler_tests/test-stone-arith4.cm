int main () {
    stone s1;
    stone s2;
    stone s3;
    stone s4;

    s1 = "14854673828914735847914672361";
    s2 = "201782412734165593248672146782593647";
    s3 = s1 + s2;
    s4 = s2 - s1;

    print_stone(s1);
    print_stone(s2);
    print_stone(s3);
    print_stone(s4);
}
