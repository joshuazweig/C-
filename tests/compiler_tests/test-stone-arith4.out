2FFF7F89D7F931B3AE5510E9
26DCA41BB7DFFC8B74DC71B846A26F
26DCA44BB75F86636E0E25669BB358
26DCA3EBB86072B37BAABE09F19186
